fast/assets/templates/workflow-gitlab.yaml

```diff
@@ -0,0 +1,167 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+default:
+  image:
+    name: registry.gitlab.com/gitlab-org/terraform-images/releases/1.1 
+
+variables:
+  FAST_OUTPUTS_BUCKET: ${outputs_bucket}
+  FAST_SERVICE_ACCOUNT: ${service_account}
+  FAST_WIF_PROVIDER: ${identity_provider}
+  SSH_AUTH_SOCK: /tmp/ssh_agent.sock
+  TF_PROVIDERS_FILE: ${tf_providers_file}
+  TF_VAR_FILES: ${tf_var_files == [] ? "''" : join("\n    ", tf_var_files)}
```

Comment:
nit: you could use [`indent`](https://www.terraform.io/language/functions/indent)

Reply:
👍

Reply:
Thanks for this. Leaving for the moment as we have the same here: https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/ab646f38ed6e0168c3fb9ea4a275dbb54e25b675/fast/assets/templates/workflow-github.yaml#L31